kettle/make_db.py

```diff
@@ -39,6 +39,7 @@ def pad_numbers(string):
     return re.sub(r'\d+', lambda m: m.group(0).rjust(16, '0'), string)
 
 WORKER_CLIENT = None  # used for multiprocessing
+GARBAGE_JOBS = ('pr-e2e-gce', 'maintenance-ci-testgrid-config-upload', 'ci-test-infra-benchmark-demo', 'ci-kubernetes-coverage-unit')
```

Comment:
`GARBAGE_JOBS` 😢

Reply:
I am happy to change to "Excluded Jobs" if we want job moral to stay high

Reply:
I was just joking, it doesn't matter to me 😄